earthpy/plot.py

```diff
@@ -71,6 +71,61 @@ def colorbar(mapobj, size="3%", pad=0.09):
     return fig.colorbar(mapobj, cax=cax)
 
 
+def _plot_image(
```

Comment:
Nice work refactoring this out into its own function!

Reply:
thank you @mbjoseph !! i had to think about it for a while and then finally realized how to implement it more efficiently! it actually makes managing the entire main function easier i just didn't know how to do it at first as my brain was making it more complicated than it needed to be!!